src/App.js

```diff
@@ -1,13 +1,13 @@
 import "./App.css";
 import React from "react";
-
+// import PropTypes from "prop-types";
```

Comment:
same here

Reply:
![category-test](https://user-images.githubusercontent.com/81489386/126899471-00085dd4-dc13-482a-a827-7f278668f35c.PNG)

to test, please import the component and give it a data prop, like this:
  let obj = {
    Image: "https://image.flaticon.com/icons/png/128/3823/3823393.png",
    id: "5",
    Title: "Veggie",
  };

<Category data={obj}/>

Reply:
can yo add this to the description of the PR